project2/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="card">
+        <div class="content">
+            <p class="moviename">Her</p>
+            <p class="rating">rating: 8.0/10</p>
+            <p class="description">R | 2h 6min | Drama, Romance, Sci-Fi | 10 January 2014 (USA)</p>
+            <p class="description">In a near future, a lonely writer develops an unlikely relationship with an operating
+                system
+                designed to meet his every need.</p>
+            <p><a href="https://www.youtube.com/watch?v=WzV6mXIOVl4">WATCH TRAILER</a></p>
+        </div>
+
+
+
+        <img src="images/her.jpg" alt="movieposter">
```

Comment:
1. `alt` text is read aloud by a screenreader, so combining words like this without spaces is confusing.
2. `alt` text is meant to replace the info that is provided by an image; here, you could describe the visuals and/or reproduce some of the text content of the image in your `alt` text.
```suggestion
        <img src="images/her.jpg" alt="Movie poster for Her: A Spike Jonze Love Story. Actor Joaquin Phoenix stares out from a vivid red background.">
```